tests/runtime/config

```diff
@@ -10,6 +10,12 @@ EXPECT_REGEX ^\s+[0-9a-f]+$
 TIMEOUT 5
 BEFORE ./testprogs/uprobe_test
 
+NAME config takes precedence over the default
```

Comment:
I'm wondering if we should also add tests specifically for `kstack`/`ustack` builtins to check that they respect the config setting (presumably to `tests/runtime/builtin`).

Reply:
Yeah, I wasn't sure where this test belonged TBH. I think you're right in that this is more specific to these builtins so I'll move the test there.